jsonschema-hyperschema.xml

```diff
@@ -854,6 +855,87 @@
+                        resource.
+                    </t>
+                    <t>
+                        Resources that are the target of a "collection" link using HTTP or an analogous
+                        protocol in JSON Hyper-Schema MUST NOT assign semantics other than resource
+                        creation to POST (or the analogous method in non-HTTP protocols).
+                    </t>
+                    <t>
+                        The <xref target="submissionSchema">"submissionSchema"</xref> field for the
+                        link SHOULD be identical to the schema of the representations of the
+                        collection's items, as indicated by the "item" link.  RFC 6573 identifies
+                        the "collection" and "item" link relation types as reciprocal, so the
+                        context resource of an "item" link MAY be treated as a collection, even if
+                        no explicit "collection" link is defined.
+                    </t>
+                    <figure>
+                        <preamble>
+                            The first of these hyper-schemas describes an individual "thing",
+                            while the second describes a collection of "things".  Note that the
+                            "targetSchema" of the individual thing's "self" link is the same
+                            as the "submissionSchema" of its "collection" link.
+                        </preamble>
+                        <artwork>
+<![CDATA[{
+    "$id": "http://example.com/schemas/thing",
+    "type": "object",
+    "properties": {
+        "id": {
+            "type": "integer",
+            "readOnly": true
+        }
+    },
+    "links": [{
+        "rel": "self",
+        "href": "/things/{id}",
+        "targetSchema": {"$ref": "#"}
+    }, {
+        "rel": "collection",
+        "href": "/things"
+        "targetSchema": {"$ref": "thing-collection"}
+        "submissionSchema": {"$ref": "#"}
+    }]
+}]]>
+
+<![CDATA[{
+    "$id": "http://example.com/schemas/thing-collection",
+    "type": "array",
+    "items": {
+        "allOf": [{"$ref": "thing"}]
+        "links": [{
+            "anchorPointer": "",
+            "rel": "item",
+            "href": "/things/{id}",
+            "targetSchema": {"$ref": "thing"}
+        }]
+    },
+    "links": [{
+        "rel": "self",
+        "href": "/things",
+        "targetSchema": {"$ref": "#"},
+        "submissionSchema": {"$ref": "thing"}
+    }]
+}]]>
+                        </artwork>
+                        <postamble>
+                            In the hyper-schema for the collection, the "item" link also demonstrates
+                            the usage of "anchorPointer", as the context of that link must be the
+                            entire collection, rather than the individual array element to which the
+                            links is attached.  The collection's self link also supports a
```

Comment:
typo: the link

Reply:
Thanks, fixed.

And yes, this is less abrupt in the reworked version, which I've been poking at.